### DISCUSSION TITLE
I set the group parameters, but it still doesn’t work

### DISCUSSION THREAD

**Discussion by YunYe-C-X** [n3d1117/chatgpt-telegram-bot#521] [Q&A]
![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/145584407/343d9cba-8012-47bb-b2ff-6a3f1d5083bd)
The GROUP_TRIGGER_KEYWORD parameter is set to bot but the robot does not reply when using "bot hello" in the group.

**Comment by yurnov** [comment]
Hi @YunYe-C-X,

did you configuried bot with send the [/setinline](https://core.telegram.org/bots/inline) command to [@BotFather](https://telegram.me/botfather)?
